js/build-utils/webpack.prod.js

```diff
@@ -1,12 +1,16 @@
 const commonPaths = require('./common-paths');
 
 const webpack = require('webpack');
-const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const path = require('path');
+const MiniCssExtractPlugin = require("mini-css-extract-plugin");
+const autoprefixer = require('autoprefixer');
 
 const config = {
   mode: 'production',
   entry: {
-    app: [`${commonPaths.appEntry}/index.js`]
+    app: [`${commonPaths.appEntry}/index.js`],
+    '../../css/react_paragraphs.field_widget': [`${commonPaths.drupalEntry}/field_widget.scss`],
+    '../../css/react_paragraphs.field_formatter': [`${commonPaths.drupalEntry}/field_formatter.scss`]
```

Comment:
does this need the full path? Doesn't the MiniCssExtractPlugin plugin config allow you to chunk out the paths you need?

Reply:
:man_shrugging: i'm just doing the best i can. it worked so i said "ok i'll leave it at that" :smile: